hawkular-component/pom.xml

```diff
@@ -51,30 +52,38 @@
       <artifactId>hawkular-metrics-bus</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-api</artifactId>
+      <version>${version.org.hawkular.accounts}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging-annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging-processor</artifactId>
+    </dependency>
     <!-- Overlay -->
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>hawkular-metrics-api-jaxrs</artifactId>
       <version>${project.version}</version>
       <type>war</type>
-      <scope>runtime</scope>
+      <scope>provided</scope>
```

Comment:
I don't recall to be honest. There may not have been. I was having trouble with some dependencies getting pulled into the WAR that should not have been there. I do not think that the runtime scope was the source of my problem. I would have to go back and check to see what if any difference runtime scope yields.